src/components/toniq-button/toniq-button.tsx

```diff
@@ -0,0 +1,16 @@
+import React, {FunctionComponent} from 'react';
+import './toniq-button.scoped.scss';
+
+export type ToniqButtonProps = Partial<{
+    text: string;
+    type: 'default' | 'selection';
+}>;
+
+export const ToniqButton: FunctionComponent<ToniqButtonProps | undefined> = (
+    props?: ToniqButtonProps,
+) => {
+    const text = props?.text ?? '';
+    const type = props?.type ?? 'default';
+
+    return <button className={`toniq-button ${type}`}>{text}</button>;
```

Comment:
Should this use `styled.button` from `styled-components` like `ToniqIcon` is doing with `styled.div`?

Reply:
while `style-components` works for you, I'll like to stick using classes as it would be much easier to manage and maintain later on with different components

Reply:
We should pick a single approach for all the components rather than following different patterns.

That being said, I actually prefer classes as well myself. I didn't know if `styled-components` was some kind of "this is the React way", so I'm happy to learn it's not 😆. I'll update `ToniqIcon` to remove its use of `styled-components`.

Reply:
I'm glad we share the same thought. I haven't used `styled-components` though but I did look into it. If you've observed way back on `hello-world` component that we created, I did changed the `styled-components` into classes.  Which I then added "scoped" css/scss. Inspecting the component that uses `styled-components` is just adding all the styles inline. That would be hard to manage and maintain. And yes I agree with the single approach. Hopefully we could discuss even more moving forward.